project_task_code/hooks.py

```diff
@@ -10,8 +10,10 @@ def pre_init_hook(cr):
     code constraint when the module is installed and before the post-init-hook
     is launched.
     """
-    cr.execute("ALTER TABLE project_task " "ADD COLUMN code character varying;")
-    cr.execute("UPDATE project_task " "SET code = id;")
+    cr.execute(
+        "ALTER TABLE project_task " "ADD COLUMN IF NOT EXISTS code character varying;"
```

Comment:
I'd keep all keywords CAPS to match styling

Reply:
Sorry, I can't quite understand you. You mean this statement should look like this: ALTER TABLE project_task ADD COLUMN IF NOT EXISTS code CHARACTER VARYING;